src/workers/notifications/utils.ts

```diff
@@ -64,8 +66,21 @@ export async function articleNewCommentHandler(
 
   const { post, source } = postCtx;
   const excludedUsers = [comment.userId];
-
   const isReply = !!comment.parentId;
+
+  if (source.type === SourceType.Squad) {
+    const comments = await con
+      .getRepository(Comment)
+      .countBy({ userId: comment.userId });
```

Comment:
Would it not be simpler to always try the insert or ignore directly?
Seems we don't particular would care about querying first right?

Also the query doesn't take specific squads in concern

Reply:
Yeah, was also thinking that initially but hesitated. Now that you mentioned it, let's do that.